notebooks/quickstart/data-scientist/02-explore-dataset.ipynb

```diff
@@ -0,0 +1,481 @@
+{
```

Comment:
<p>We can now skip calling .exists, instead just call the pointer. Update to latest version of syft to see this work.</p><p> </p><p>If we check the mean_age variable, we can learn three properties about this data.</p><ul><li>PointerId: It is the unique id for this pointer</li><li>Status (Ready/Processing): This indicates if the value for this pointer has been calculated on the server side.</li><li>Representation: Shows a synthetic representation of what value this data can represent.</li></ul><pre class="ql-syntax" spellcheck="false">mean_age
</pre>
<br/>

 _Reply via <a href='https://app.reviewnb.com/OpenMined/PySyft/pull/7045/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='34'/>